googletest/Makefile.am

```diff
@@ -120,16 +121,16 @@ EXTRA_DIST += \
 
 # MSVC project files
 EXTRA_DIST += \
-  msvc/gtest-md.sln \
-  msvc/gtest-md.vcproj \
-  msvc/gtest.sln \
-  msvc/gtest.vcproj \
-  msvc/gtest_main-md.vcproj \
-  msvc/gtest_main.vcproj \
-  msvc/gtest_prod_test-md.vcproj \
-  msvc/gtest_prod_test.vcproj \
-  msvc/gtest_unittest-md.vcproj \
-  msvc/gtest_unittest.vcproj
+  msvc/2010/gtest-md.sln \
```

Comment:
I would avoid adding vusual studion 2010 (!) ... why?

Reply:
The files got moved from `msvc/*` to `msvc/2010/*` at some point, but `Makefile.am` was not changed at the same time.  If you think it would be best to remove the files altogether, then sure, but shouldn't that be a separate change?

Reply:
Since we are making a change, I just want to understand why. Could you please take a look and suggest why this happened and more importantly is this the general enough way to make this change for all googletest users?

Reply:
@coryan

Reply:
> Since we are making a change, I just want to understand why.
> Could you please take a look and suggest why this happened

**TL;DR;** since there was no test for automake builds, nothing detected the breakage.

Let's play code archeologist:

The last time those lines in `Makefile.am` were touched was around 9 years ago:

https://github.com/google/googletest/blame/master/googletest/Makefile.am

At that time the `msvc/*` directory was full of files:

https://github.com/google/googletest/tree/bcaf6f542fac67549528027c24e530e0e89f700b/msvc

The files were removed in this commit:

https://github.com/google/googletest/commit/45287f3dffa494d612c4ba94199aeaa40c99571f

Which I think you merged here:

https://github.com/google/googletest/commit/f25921565bbfb16b02919165669734a5bea7ee98


> and more importantly is this the general enough way to make this change for all
> googletest users?

Well, those particular lines in `Makefile.am` are simply there to include the MSVC files in any tarballs created for a "distribution".  Whether you want to continue to support tarballs or not is a different question.  Right now automake insists on creating the tarball and testing the tarball.  I think if you want to drop support for tarballs that should be a separate change.  Or you may want to drop the MSVC files altogether, that is an option too, but those should be separate changes from "getting automake to work again".

Reply:
makes sense!